src/Manager/ProviderManager.php

```diff
@@ -41,6 +48,11 @@ public function all(): array
         return $this->providers;
     }
 
+    /**
+     * @param string $code
+     *
+     * @return bool
+     */
```

Comment:
This phpdoc belongs in the interface

Reply:
The whole method is missing from the interface. I'll add :)